grades_api/api.py

```diff
@@ -0,0 +1,65 @@
+from opaque_keys.edx.keys import CourseKey
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.response import Response
+from django.contrib.auth.models import User
+from django.db import transaction
+from openedx.core.lib.api.permissions import ApiKeyHeaderPermission
+from courseware import courses
+from courseware.grades import grade
+
+def _computed_passed(grade_cutoffs, percent = 0.0):
+    """
+    Computes and returns whether the given percent value
+    is a passing grade according to the given grade cutoffs.
+    """
+    nonzero_cutoffs = [cutoff for cutoff in grade_cutoffs.values() if cutoff > 0]
+    success_cutoff = min(nonzero_cutoffs) if nonzero_cutoffs else None
+    return success_cutoff and percent >= success_cutoff
```

Comment:
Not sure about priority between "and" and ">=":
```suggestion
    return success_cutoff and (percent >= success_cutoff)
```